torch/csrc/jit/script/python_sugared_value.cpp

```diff
@@ -544,6 +544,17 @@ std::shared_ptr<SugaredValue> toSugaredValue(
   } else if (
       obj.ptr() == py::module::import("torch.jit").attr("annotate").ptr()) {
     return SpecialFormValue::create(prim::annotate);
+#ifdef USE_DISTRIBUTED
+  } else if (
+      // RPC module is only avaialble for Python3
+      // when build flag "USE_DISTRIBUTED" is on.
+      !py::module::import("torch._six").attr("PY2").cast<bool>() &&
+      obj.ptr() ==
+          py::module::import("torch.distributed.rpc.api")
```

Comment:
what's `api` here for? I thought user facing api would be `torch.distributed.rpc.rpc_async`

Reply:
They actually point to the same function object. But let me change it to rpc, to make the code easier to understand.